mlflow/store/db_migrations/versions/1bd49d398cd23_add_secrets_tables.py

```diff
@@ -176,8 +176,25 @@ def upgrade():
         ),
     )
 
+    op.create_table(
```

Comment:
What happens for people who already have revision '1bd49d398cd23' in the database? Do they need to rollback and migrate again? I'm wondering if we should separate a migration file or not in this case.

Reply:
Still safe to edit since we haven't released this yet :)